azext_iot/iothub/params.py

```diff
@@ -249,6 +251,78 @@ def load_iothub_arguments(self, _):
             help="MIME Type of file.",
         )
 
+    with self.argument_context("iot hub state") as context:
+        context.argument(
+            "state_file",
+            options_list=["--state-file", "-f"],
+            help="The path to the file where the state information will be stored."
+        )
+        context.argument(
+            "replace",
+            options_list=["--replace", "-r"],
```

Comment:
this was worse before. There are multiple options so far:
- replace (r)
- clean (c)
- overwrite (o)

I will keep replace since it feels more correct; you will replace all aspects of a hub. We can have a poll too

Reply:
I'm not a fan of 'o' because its overloaded to mean 'output' for me ....state functionality will be first introduced as preview so we can see if we get any feedback on this.